### PR TITLE
 Flexlayout

### PR DESCRIPTION
**Pull request by shivanipitta**

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -7,33 +7,31 @@ import 'hammerjs';
...
 
 
-
 @NgModule({
   imports: [BrowserModule, FormsModule, ReactiveFormsModule, BrowserAnimationsModule,
     MatButtonModule, MatCheckboxModule, MatFormFieldModule, MatInputModule,
-    MatOptionModule, MatSelectModule, MatSlideToggleModule, MatCardModule, MatIconModule,
-    MatChipsModule, MatToolbarModule, MatIconModule, MatListModule, RouterModule.forRoot(appRoutes)],
+    MatOptionModule, MatSelectModule, MatSlideToggleModule, MatCardModule,
+    MatChipsModule, MatToolbarModule, MatIconModule, MatListModule, FlexLayoutModule, RouterModule.forRoot(appRoutes)],
```
Comment (joebochill) [inline src/app/app.module.ts]: Are we not using MatIconModule?

File: src/app/registration/registration.component.scss [hunk 2]
```diff
@@ -55,22 +55,14 @@
   margin: 0 auto 10px;
   display: block;
 }
-
-.flex-center {
-  display: flex;
-  align-items: center;
-    justify-content: space-between;
-  
+.resend-group{
```
Comment (joebochill) [inline src/app/registration/registration.component.scss]: Lines 1-8 in registration.component.scss:  are we using these?

Comment (shivanipitta) [reply src/app/registration/registration.component.scss]: yes, we are using [flex], [flex-1]

File: src/app/app.module.ts (hunk 1 above)
Comment (shivanipitta) [reply src/app/app.module.ts]: we are using MatIconModule

**Pull request merged**